glossary/ansible-openwisp2.rst

```diff
@@ -0,0 +1,27 @@
+=====================
+Technologies Involved
+=====================
+
+- `Python<https://www.python.org/>`_
```

Comment:
please leave one space before `<` and after `Python`, same applies for the rest of the terms